src/reducers/__tests__/deleteThought.ts

```diff
@@ -80,3 +80,20 @@ it('update cursor after thought deletion', () => {
     },
   ])
 })
+it('deletion of empty thought', () => {
+  const steps = [newThought(''), newThought('')]
+
+  const state = initialState()
+  const stateNew = reducerFlow(steps)(state)
+  const rootChildren = stateNew.thoughts.thoughtIndex[HOME_TOKEN].children
```

Comment:
Never access `state.thoughts` directly since it is a low-level implementation detail. Instead, use a selector like `getAllChildren`.